src/components/Attachments/PreviewModal.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useRef } from "react";
+
+import saveAs from "file-saver";
+import { findIndexBy } from "neetocommons/pure";
+import { Download, LeftArrow, RightArrow } from "neetoicons";
```

Comment:
```suggestion
import { Download, Left, Right } from "neetoicons";
```

`Left` and `Right` icons have better aesthetics.